editor/plugins/skeleton_ik_editor_plugin.cpp

```diff
@@ -42,20 +42,17 @@ void SkeletonIKEditorPlugin::_play() {
 
 	if (play_btn->is_pressed()) {
 
-		initial_bone_poses.resize(skeleton_ik->get_parent_skeleton()->get_bone_count());
-		for (int i = 0; i < skeleton_ik->get_parent_skeleton()->get_bone_count(); ++i) {
-			initial_bone_poses.write[i] = skeleton_ik->get_parent_skeleton()->get_bone_pose(i);
-		}
+		initial_bone_count = skeleton_ik->get_parent_skeleton()->get_bone_count();
 
 		skeleton_ik->start();
 	} else {
 		skeleton_ik->stop();
 
-		if (initial_bone_poses.size() != skeleton_ik->get_parent_skeleton()->get_bone_count())
+		if (initial_bone_count != skeleton_ik->get_parent_skeleton()->get_bone_count())
```

Comment:
Since we are now using `pose_override` we don't need any more to store the bone transforms, so we can take rid of this check and `initial_bone_count` completely.